### PR TITLE
Create Autolab configure tabs

### PR DESCRIPTION
**Pull request by najclark**
## Description
Creates button under 'Manage Autolab' to configure Autolab features such as LTI, SMTP, Github, and OAuth.

## Motivation and Context
Instead of having to configure autolab through yml files and other config files, this change provides a UI to do so.

## How Has This Been Tested?
Click 'Manage Autolab', click 'LTI', update LTI settings, and confirm the window stays on the LTI config page.
(See https://github.com/autolab/Autolab/pull/1769 for LTI settings to use)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have run rubocop for style check. If you haven't, run `overcommit --install && overcommit --sign` to use pre-commit hook for linting
- [x] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -65,12 +65,12 @@
...
       <li role="presentation"><%= link_to "Manage Users", users_path, title: "Manage every Autolab user" %></li>
 
-      <li role="presentation"><%= link_to "Github Integration", github_integration_admin_path, title: "Manage Github Integration" %></li>
-
       <li role="presentation"><%= link_to "Manage API Applications", oauth_applications_path, title: "Manage registered API clients" %></li>
 
       <li role="presentation"><%= link_to "Bulk Email Instructors", email_instructors_admin_path, title: "Send a bulk email to every Autolab instructor" %></li>
 
+      <li role="presentation"><%= link_to "Configure Autolab", autolab_config_admin_path, title: "Configure Autolab settings" %></li>
+
```
Comment (victorhuangwq) [inline app/views/layouts/_navbar.html.erb:73]: It's likely not related to your PR - but why is there dropdown2 and dropdown3?

File: spec/controllers/admins_controller_spec.rb [hunk 2]
```diff
@@ -33,4 +33,76 @@
...
+          if File.exist?("#{Rails.configuration.lti_config_location}/lti_tool_jwk.json")
+            File.delete("#{Rails.configuration.lti_config_location}/lti_tool_jwk.json")
+          end
+          if File.exist?("#{Rails.configuration.lti_config_location}/lti_platform_jwk.json")
+            File.delete("#{Rails.configuration.lti_config_location}/lti_platform_jwk.json")
+          end
+        end
+      end
+
+      context "when user is Instructor" do
```
Comment (victorhuangwq) [inline spec/controllers/admins_controller_spec.rb:77]: nit: instructor instead of Instructor to be consistent

File: config/lti_platform_jwk.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{"kty":"RSA",
+  "e":"AQAB",
+  "n":"1-ACVFVSZ6ee2Iv2A3S4O-fcrz4ki3Sigh-1vQcCCbi1arb5najJby9r0rrHbgqpTVM7_LLGvQz8siLUBKdvRAQo6tC0zsI-sbNn4Nu1kK339gk83w56DdqDNsPJohb2Zmkc3GOZYVTncIA9AGrdA2l0R64nWK2gl8y948bRIj9r0fzWgwlIAVwHXmlMoLNBZ6MdcaX85pia_gLBKp4ZFZNF8zWms8h2k9f08AbB2bME04jTfjGOIsHrPYWDfzGyHAueKV9000nzRgQ0LS-ponwXjXZ1Ocqu86sxws19g75TBrpoz-GRODXkwRTJBenlKI-Va7oyUEQZ6jYmgmdFwQ",
```
Comment (victorhuangwq) [inline config/lti_platform_jwk.json:3]: Similarly, this file as well

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
Correct me if I'm wrong, but I don't think the json files should be part of the changes. Tested and everything else looks good to me

File: app/views/layouts/_navbar.html.erb (hunk 1 above)
Comment (najclark) [reply app/views/layouts/_navbar.html.erb:73]: I think it's related to supporting mobile browsers, not 100% sure though.

File: config/lti_platform_jwk.json (hunk 3 above)
Comment (najclark) [reply config/lti_platform_jwk.json]: Removed

**Review by victorhuangwq — APPROVED** [review]
LGTM!

**Pull request merged**